incubator/uchiwa/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+Getting Started:
+
+1. Uchiwa can be accessed at the following address from within your cluster: http://{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.httpPort }}
```

Comment:
I'd say implementing that would be the right step, otherwise add instructions for how to use kubectl proxy to access it (also shown in the Jenkins chart).